vllm/config.py

```diff
@@ -333,8 +335,8 @@ def get_num_kv_heads(self, parallel_config: "ParallelConfig") -> int:
 
     def get_num_attention_heads(self,
                                 parallel_config: "ParallelConfig") -> int:
-        return self.hf_text_config.num_attention_heads // \
-            parallel_config.tensor_parallel_size
+        num_heads = getattr(self.hf_text_config, "num_attention_heads", 0)
+        return num_heads // parallel_config.tensor_parallel_size
```

Comment:
Oh it seems `get_num_attention_heads` can be 0 sometimes, curious about when will it be the case?

Reply:
Our speculator models do not use attention :)